src/Components/NavBar.css

```diff
@@ -0,0 +1,39 @@
+/* stylelint-disable selector-type-case */
+/* stylelint-disable selector-type-no-unknown */
```

Comment:
[Optional] Kindly avoid disabling inline linter checks as they help you maintain best coding practices and share coding standards with your team.

Reply:
Thanks for the review.
We will take that into consideration